core/src/main/scala/pickling/Pickler.scala

```diff
@@ -84,6 +86,22 @@ trait Unpickler[T] {
 }
 object Unpickler {
   def generate[T]: Unpickler[T] = macro Compat.UnpicklerMacros_impl[T]
+  implicit def spuToU[T: SPicklerUnpickler]: Unpickler[T] = implicitly[SPicklerUnpickler[T]].unpickler
+}
+
+/** A combination of a static pickler and an unpickler for type `T`.
+ */
+trait SPicklerUnpickler[T] {
```

Comment:
If `SPicklerUnpickler[T]` extends `SPickler[T]` and `Unpickler[T]` should the picklers shipped with `Defaults` use them too? In other words, differentiates custom-written picker/unpicklers vs built-in pickler/unpicklers?